web-api/terraform/modules/opensearch-sync/opensearch-sync.tf

```diff
@@ -0,0 +1,59 @@
+data "aws_caller_identity" "current" {}
+
+module "opensearch_sync_lambda" {
+  source         = "../lambda"
+  handler_file   = "./web-api/src/lambdas/opensearch/sync-handler.ts"
+  handler_method = "syncHandler"
+  lambda_name    = "opensearch_sync_lambda_${var.environment}_${var.color}"
+  role           = var.lambda_role_arn
+  environment    = var.lambda_environment
+  timeout        = "900"
+  memory_size    = "3008"
+}
+
+resource "aws_sqs_queue" "opensearch_sync_queue" {
+  name                       = "opensearch_sync_queue_${var.environment}_${var.color}"
+  visibility_timeout_seconds = 901
+  redrive_policy = jsonencode({
+    deadLetterTargetArn = aws_sqs_queue.opensearch_sync_dl_queue.arn
+    maxReceiveCount     = 1
+  })
+}
```

Comment:
Hmm. I think we assumed that this would preserve message order, but SQS only preserves message order if it is set up at creation as a FIFO Queue. See [here](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_CreateQueue.html). We probably need to do this?